common/src/main/java/org/conscrypt/OpenSSLAeadCipher.java

```diff
@@ -220,15 +220,6 @@ boolean allowsNonceReuse() {
         return false;
     }
 
-    /**
```

Comment:
What does linter complain about here?

Reply:
The two `@throws` with no details.  However we generally don't duplicate Javadoc from a base class so this shouldn't really be here anyway (I think my bad for missing it in a code review).